### PR TITLE
docs(connector): add github readme zh-CN

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add GitHub readme zh-CN.

## Testing

N/A

### CODE REVIEW

File: packages/connector-github/README.md [hunk 1]
```diff
@@ -1,32 +1,85 @@
...
+You can leave **Application description** field blank and customize **Authorization callback URL** as `${your_logto_origin}/callback/github-universal`.
 
-## Managing GitHub OAuth apps
+#### Managing OAuth apps
 
 Go to the [OAuth Apps page](https://github.com/settings/developers) and you can add, edit or delete existing OAuth apps.
 You can also find `Client ID` and generate `Client secrets` in OAuth app detail pages.
 
-As for managing GitHub Apps we mentioned in the last section, go to the [GitHub Apps page](https://github.com/settings/apps).
+#### Create and configure GitHub app
```
Comment (gao-sun) [inline packages/connector-github/README.md:60]: Didn't follow this. pending

**Review by gao-sun — CHANGES_REQUESTED** [review]

File: packages/connector-github/README.md [hunk 2]
```diff
@@ -1,32 +1,55 @@
...
+  - [创建并配置 OAuth 应用程序](#创建并配置-oauth-应用程序)
+  - [管理 OAuth 应用程序](#管理-oauth-应用程序)
+  - [编写连接器的 JSON](#编写连接器的-json)
+    - [配置类型](#配置类型)
+  - [测试 GitHub 连接器](#测试-github-连接器)
+  - [参考](#参考)
+
+## Get started
+
+The GitHub connector enables end-users to have access to your application using their own GitHub accounts via GitHub OAuth 2.0 authentication protocol.
```
Comment (gao-sun) [inline packages/connector-github/README.md:30]:
```suggestion
The GitHub connector enables end-users to sign in to your application using their own GitHub accounts via GitHub OAuth 2.0 authentication protocol.
```

File: packages/connector-github/README.md [hunk 3]
```diff
@@ -43,3 +66,70 @@ Here is an example of GitHub connector config JSON.
...
+## Reference
+
+- [GitHub - Developers - Apps](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps)
+- [GitHub - Developers - Apps - Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
+
+# GitHub 连接器
+
+## 开始上手
+
+GitHub 连接器让终端用户可以用 GitHub 账号，通过 GitHub OAuth 2.0 授权协议来访问你的应用。
```
Comment (gao-sun) [inline packages/connector-github/README.md:84]:
```suggestion
GitHub 连接器让终端用户可以用 GitHub 账号，通过 GitHub OAuth 2.0 授权协议来登录你的应用。
```

File: packages/connector-github/README.md [hunk 4]
```diff
@@ -43,3 +66,70 @@ Here is an example of GitHub connector config JSON.
...
+## 编写连接器的 JSON
+
+用你从上一步 OAuth 应用程序详情页获取到的 _Client ID_ 和 _Client Secret_ 分别填写 `clientId` 和 `clientSecret`。
+
+以下是一个 GitHub 连接器配置 JSON 的样例。
+
+```json
+{
+    "clientID": "<your-client-id>",
+    "clientSecret": "<your-client-secret>"
```
Comment (gao-sun) [inline packages/connector-github/README.md:116]:
```suggestion
  "clientID": "<your-client-id>",
  "clientSecret": "<your-client-secret>"
```

Comment (gao-sun) [reply packages/connector-github/README.md:116]: ditto for English

**Review by gao-sun — APPROVED** [review]
great

File: packages/connector-github/README.md (hunk 1 above)
Comment (darcyYe) [reply packages/connector-github/README.md]: Discussed offline, should only leave OAuth Apps in the README guide. GitHub Apps is beyond this scope.

**Pull request merged**